rfcs/077-enriching-addressables-with-works/README.md

```diff
@@ -0,0 +1,161 @@
+# RFC 077: Content API: Enriching addressable content types with Works data
+
+We'd like to enrich the content pages of our website, i.e. those whose data is retrieved from Prismic, with previews of Works that are linked to within them. This RFC proposes augmenting the [Addressable content types](https://github.com/wellcomecollection/docs/tree/main/rfcs/062-content-api-all-search#addressable-content-types) with Works data and creating a new endpoint for retrieving individual Addressable items, so that we can create the desired UI with a single API call.
+
+**Last modified:** 2025-07-07T15:15:00+00:00
+
+## Table of contents
+
+- [Background information](#background-information)
+- [Proposal](#proposal)
+- [Integration with the front-end](#integration-with-the-front-end)
+- [Alternatives](#alternatives)
+
+## Background information
+
+We wish to provide previews of the Works linked to from our content pages. See ['Works recap in this article (cards at end of article)'](https://github.com/wellcomecollection/wellcomecollection.org/issues/12065) and ['Work links in Prismic content: Preview component'](https://github.com/wellcomecollection/wellcomecollection.org/milestone/86)
+
+The main focus for this work to date has been Stories, but other content types can include links to Works. For example, they currently appear in Exhibitions, Events, Guides, Articles and Pages, and there is nothing preventing them from being added to other Addressable Prismic content types in the future. We'd like to provide the same UI for any content types that include Works links.
+
+We need to:
+
+- Account for all the possible Addressable Prismic content types
+- Retrieve the data to display the previews with as few requests as possible
+- Retrieve only the data necessary to display the previews
+- Keep using Prismic's preview functionality, so we will still render the main content of the page using Prismic's API
+
+We already have all the [Addressable content types in an Elasticsearch index, which we use for the 'all' search](https://github.com/wellcomecollection/docs/blob/main/rfcs/062-content-api-all-search/README.md).
+
+- We have an `https://api.wellcomecollection.org/content/v0/all/` endpoint which returns an ordered list of Addressable content types.
+
+## Proposal
+
+- Augment the Addressable content types with data from any Works they reference
+- Create a new endpoint for retrieving individual Addressable items
+
+### Proposed model for Work data
+
+We would add a `works` property to the `display` property of each of the indexed Addressables. The `works` property would contain an array of Works. Each Work would have the following data taken directly from the Catalogue API response for a Work:
```

Comment:
`relatedWorks`? what do you think?

Reply:
I'd think they were related to the story 🤔 but happy with Linked Works, it describes it best!